hat/assets/js/apps/Iaso/components/dialogs/ImageGalleryComponent.tsx

```diff
@@ -134,6 +140,7 @@ const ImageGallery: FunctionComponent<Props> = ({
                     >
                         <Close sx={styles.closeIcon} />
                     </IconButton>
+                    {link}
```

Comment:
Following other comments:
I would extract and rename your `FormSubmissionLink`component, call it here and pass it a `url` that has been received in the props (instead of the current `link` prop).